lib/order.rb

```diff
@@ -0,0 +1,29 @@
+require_relative 'menu'
+require_relative 'takeaway'
+
+class Order
+  attr_reader :basket, :menu
+
+  def initialize
```

Comment:
Could we use dependency injection here to decouple the classes? https://github.com/makersacademy/skills-workshops/blob/f5b4801840fe07d26ff70341652dc81dcda12289/practicals/object_oriented_design/dependency_injection.md